Update syllabus.md (#23)
mids-w203/syllabus: syllabus.md

- There is a required course packet, which you may purchase at study.net.  This includes select chapters from other textbooks, namely *Probability and Statistics, 8th Edition* by Devore. 

### Optional Textbooks
The following textbooks are not required but are available for those who would like extra practice problems and/or additional exploration of course concepts. Both are available through the UC library

- _Modern Mathematical Statistics with Applications_ by Devore and Berk ([library link](https://link-springer-com.libproxy.berkeley.edu/book/10.1007%2F978-1-4614-0391-3)) is very readable and has lots of worked examples. It's a good complement to Aronow and Miller's concise mathematical language.

> - _Elementary Probability and Applications_ by Durrett ([library link](https://ebookcentral-proquest-com.libproxy.berkeley.edu/lib/berkeley-ebooks/detail.action?docID=461127)) has a lot of examples for you to work through. It's mostly examples. Only one person can have access to this ebook at a time, so to avoid blockages consider buying a physical textbook for lik $30 on [amazon](https://www.amazon.com/Elementary-Probability-Applications-Rick-Durrett-dp-0521867568/dp/0521867568/ref=mt_other?_encoding=UTF8&me=&qid=). 

Reviewer: Lik --> like

Reviewer 2: I thought we found another way for students to access the Durrett book without the one-person-at-a-time problem?

Author: How... do... I... just... add... a... single... letter... from... this... review... interface..?

Author: ```suggestion
- _Elementary Probability and Applications_ by Durrett ([library link](https://ebookcentral-proquest-com.libproxy.berkeley.edu/lib/berkeley-ebooks/detail.action?docID=461127)) has a lot of examples for you to work through. It's mostly examples. Only one person can have access to this ebook at a time, so to avoid blockages consider buying a physical textbook for about $30 on [amazon](https://www.amazon.com/Elementary-Probability-Applications-Rick-Durrett-dp-0521867568/dp/0521867568/ref=mt_other?_encoding=UTF8&me=&qid=). 
```

| C-           | 70 <= x < 73 |
| Lower Grades | x < 70       |

## Participation Policy

> Participation grades are assigned on participation in live session. It is expected that you will attend the entirety of your live sesson and regularly participate in class discussion. These grades are assigned on a pass/fail basis. If you attend less than 50% of live sessions (don't show up at all to the session or drop off early) you will recieve a zero for your participation grade. Otherwise, you will get full credit for these points.

Reviewer 2: I agree with alex that we don't want such a narrow / mechanical definition of participation.  I also would rather not overexplain what it means to participate.  Here's my attempt at a revision:
```suggestion
Participation is more than just showing up to class.  It includes being prepared, engaging actively, and treating others with respect to sustain a thriving learning environment.
```

## Academic Misconduct Policy
This section covers acceptable and unacceptable conduct for the course. If you have any questions reach out to a member of the w203 intsructional staff.

We treat academic misconduct seriously and will refer cases of academic misconduct to the Center for Student Conduct at UC Berkeley per the [Student Code of Conduct](https://conduct.berkeley.edu/code-of-conduct/). This is a long drawn out process that involves going to hearings and can result in consequences up to and including suspension from the program and revocation of your degree. Don't do it. It's not worth it. 

### Acceptable Resources
- [Wolfram Alpha](https://www.wolframalpha.com) is an online tool that can be used to solve all kinds of difficult numerical manipulations like differentiation and integration. You can use this on the tests and homework as you please (_this isn't a calculus class_) but when you use it, provide a reference so that your graders know how you got from one stage to the other. 
- [Stack Overflow](https://www.stackoverflow.com) and other stack exchange sites. (_as long as you do not ask exact questions from the course_)

> - **Your fellow students** are invariably excellent individuals who will understand different aspects of statistics than you. Get together form study groups. As long as there's no copying and pasting you're fine.

Reviewer 2: ```suggestion
- **Your fellow students** are invariably excellent individuals who will understand different aspects of statistics than you. You are allowed to discuss homework with other students, but not tests.  For homework problems, strategize with other students as much as you like, but write down the people you work with on your submission, and write down your final solution yourself (do not copy from another student).
```

> - Office hours! Between the section instructors and the TA's there are about a dozen office hours a week. Usually we'll end up covering the homework. It's worth your time to go.

Reviewer 2: Worried the last sentence might come across as talking down to students.  
```suggestion
- Office hours! Between the section instructors and the TA's there are about a dozen office hours a week. Usually we'll end up covering the homework.  A lot of great learning happens during these hours and we strongly recommend them.
```

> ### Unacceptable Resources

Reviewer 2: Epic.  thank you for writing this clearly.

- Chegg, Slader, CourseHero and similar resources where students post exact questions from the course which are answered by professional "tutors" or students at other universities are **unacceptable in all circumstances**. The instructional team regularly looks at solutions posted to these websites and if we notice solutions that are substantially similar to these sites we will refer the offending student to the Center for Student Conduct.
- Copying from other students homework, websites, old solutions, etc. is cheating and offenders will be reffered to the Center for Student Conduct.

## Late Policy 

> We set deadlines for homework, tests, and labs to be completed. These deadlines are to your advantage, becacuse they help you to prioritize your learning. At the same time, we understand that MIDS a single facet of the life that you live. 

Reviewer 2: the middle sentence feels somewhat patronizing to me.
```suggestion
We set deadlines for homework, tests, and labs to be completed. At the same time, we understand that MIDS is a single facet of the life that you live, and some delays are unavoidable. 
```

- **Homework**: Homework assignments are chances for you to apply the content that you have learned in the async and the live sessions. These occur at a regular (nearly weekly) pace. **Homework is due _at the time that your live session meets_ on the day that it meets.** 
  - Students have **five "late days" that they can use without penalty** on homework through the semester. After those "late days" have been used, each day late will be assessed a 10% penalty from the final grade of that homework assignment. As an example, after using all five late days, a homework assignment turned in one day late could earn a maximum score of a 90% _on that homework_. 
  - Students **cannot turn in any single homework more than two days late.** This balances the need for student flexibility, with the need to release feedback to other students in a section. For example, a student who has class on Monday at 4:00p cannot turn in an assignment later than Wednesday at 4:00p.

>   - _Note on Gradescope due dates:_ Gradescope is the online platform that we use for homework. The due dates shown on gradescope are there to accomodate the last section of the week. You still must turn in your assignments before your live session meets. Lateness will be evaluated at the end of the quarter. When final grades are tallied. You must track how many late days you have used

Reviewer 2: I think we technically have trimesters?  I feel like most folks in the program use semester though.  
```suggestion
  - _Note on Gradescope due dates:_ Gradescope is the online platform that we use for homework. The due dates shown on gradescope are there to accomodate the last section of the week. You still must turn in your assignments before your live session meets. Late days will be counted at the end of the semester.  You must keep your own tally of how many late days you have used.
```
